engine/channels.go

```diff
@@ -51,3 +53,15 @@ const (
 func FullyQualifiedChannelName(channelID string, rootBlockID string) string {
 	return fmt.Sprintf("%s/%s", channelID, rootBlockID)
 }
+
+// ChannelConsensusCluster returns a dynamic cluster consensus channel based on
+// the chain ID of the cluster in question.
+func ChannelConsensusCluster(clusterID flow.ChainID) string {
```

Comment:
and we don't care about adding the `rootBlockID` to the channel name since the cluster id itself would be unique for each epoch, correct?

Reply:
It is unique per epoch 👍. Is that easy to add here?

Reply:
https://github.com/onflow/flow-go/pull/36

Reply:
I can merge to master after you merge or you can merge it to your branch whatever is more convenient for you :)